js/functions.js

```diff
@@ -0,0 +1,68 @@
+/*
+Иван, добрый день! Подскажите, такой вариант первой функции будет верный? :
+const getLengthCheck = (string, maxLength) => { (string.length <= maxLength) ? true : false};
+Проверял в консоли Chrome и всегда возвращалось "undefined".
+Работало только так: const getLengthCheck = (string, maxLength) => { (string.length <= maxLength) ? console.log(true) : console.log(false)};
+*/
+//Функция для проверки длины строки:
+const getLengthCheck = (string, maxLength) => {
+  if (string.length <= maxLength) {
+    return true;
+  }
+  return false;
+}
+
+//Функця для проверки, является ли строка палинтромом + случай, когда в строке встречаются пробелы:
+const getpalindromCheck = (string) => {
+  let newString = string.toLowerCase();
+    newString = newString.replaceAll(' ', '');
```

Comment:
Оба изменения строки можно зачейнить друг за другом (т.е. у строки вызовется метод `toLowerCase()` затем у полученного результата (тоже строки) вызовется метод `replaceAll()` и уже полученный результат сразу запишется в переменую `newString`) и использовать `const` 
```suggestion
    const newString = string.toLowerCase().replaceAll(' ', '');
```

На будущее, крайне желательно заывать переменные по их содержимому, `newString` -- не говорит о том, что в ней модифицированный вариант исходной строки, вот `mirrorString` -- сразу говорит нам о том, что там будет зеркальная строка, это хорошее название. Это не критично, но очень поможет в больших проектах.